app/views/notifications/_nav_menu.html.erb

```diff
@@ -1,9 +1,9 @@
 <nav class="notification-nav">
   <% if @organizations.present? %>
-    <header class="pl-2 mb-2">
-      <h4>
+    <header>
```

Comment:
One last extra piece of feedback: IF `<header>` doesn't have any extra styling, we can drop it from the markup.